### ISSUE TITLE
Problems with implementation of the code.

### ISSUE THREAD

**Issue by C8620** [K024/bangbangboom-game#6]
First of all, thank you for your work. It is a very amazing solution and is of great significance and importance as far as I am aware of.

However, I have spend literally hours trying to figure out how to at least make a duplicate of current implementations and looking for places went wrong through trials and errors and have been missing probably the most important point of all.

Would you mind explain the _actural_ implementation of this player in when trying to call out the javascript in HTML code. I have compiled a bundle using Webpack 4 but I really don't understand where come the BangGame.Game(). Obviously it is not previously defined and browser console is throwing errors regarding this.

Many thanks.

**Comment by c86-moe** [comment]
Hello there and here is some update…

After referencing previous codes, I am now able to come up with a single js file that include BangGame.Game() but console is still throwing errors of invalid configuration. Any solutions?

Many thanks.

**Comment by C8620** [comment]
Sorry for all the trouble.

Currently the problem is that it will report problem in comsole:
`Access to XMLHttpRequest at 'object:///rawmap' from origin '(my site)' has been blocked by CORS policy` (using bangbangboom-game.js from bbb-editor and the one I compiled)
Screenshot:
![image](https://user-images.githubusercontent.com/73808043/119227675-af1b4380-bb41-11eb-93ee-780b11d4320f.png)

Would you mind help solving this problem?

Many thanks.

**Comment by K024** [comment]
@c86-moe This problem may caused by the error above it. The raw map object didn't pass the schema validation. That caused the resource loader falling back to load the raw map object from network, which is impossible. 

Anyway, bbb-editor and bbb-game are out of maintenance. You can check out an active branch [here](https://gitee.com/ReiKohaku/ebbb), or have a look at new projects like [sonolus](https://sonolus.com/) and [banground](https://banground.live/)
